services/app-web/src/pages/App/AppBody.tsx

```diff
@@ -33,7 +33,7 @@ export function AppBody({
                 {loginStatus === 'LOADING' ? (
                     <Loading />
                 ) : (
-                    <AppRoutes authMode={authMode} />
+                    <AppRoutes authMode={authMode} setAlert={setAlert} />
```

Comment:
Can we name this prop `setGlobalAlert` or `setAppAlert` just to clarify what level this alert is occurring at? 

Also question - are you envisioning that is how this prop would be used in the future- by any app wide alert that' s not in a specific form ? I could also see us using a context for this - to store an app wide alert.

Reply:
I copied the pattern for catching errors in logout that we find in Header for the AuthenticatedRouteWrapper, so I needed to pass setAlert down to the wrapper.  This solution implies no vision whatsoever 🙂 .